kubos/use.py

```diff
@@ -0,0 +1,61 @@
+# Kubos SDK
+# Copyright (C) 2016 Kubos Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import git
+import sys
+
+from kubos.utils.sdk import *
+from kubos.utils.git_common import *
+from kubos import versions
+from yotta import target
+from yotta.options import parser
+
+
+def addOptions(parser):
+    parser.add_argument('set_version', nargs=1, help='Set a specific version of the KubOS modules to build your projects against.')
+
+def execCommand(args, following_args):
+    args = vars(args)
+    version = args['set_version'][0]
+    kubos_repo, origin = get_repo(KUBOS_SRC_DIR)
+    check_provided_version(version, kubos_repo)
+
+def check_provided_version(requested_version, repo):
+    #the repo paramenter allows this function to be used for the example project as well
+    active_version = get_active_kubos_version()
```

Comment:
`get_active_version` is also used in the `version` and `versions` command. I do agree these functions should be together.

Do you think that `get_active_kubos_version` should be moved down here or should `set_active_version` be moved up with the `get_active_kubos_version` function?